.travis.yml

```diff
@@ -2,6 +2,9 @@ language: r
 sudo: false
 cache: packages
 
+env:
+  - R_RVERSIONS_CRAN_URL='cloud.r-project.org'
```

Comment:
Don't we need the `https://`?

Reply:
apparently not, it worked :joy: I added it.